cmd/create.go

```diff
@@ -29,8 +30,12 @@ var componentCreateCmd = &cobra.Command{
 
 If component name is not provided, component type value will be used for the name.
 
+By default, builder images will be used from the current namespace.
+You can explicitly supply a namespace by using: odo create namespace/name:version
+If version is not specified by default, latest wil be chosen as the version.
```

Comment:
Make this into one-line, example:

```sh
By default, builder images will be used from the current namespace. You can explicitly supply a namespace by using: odo create namespace/name:version
 ```

Other than that, this PR LGTM! Once you've made the above change, feel free to merge.

Reply:
Ack